src/content/contribute/writing-a-loader.md

```diff
@@ -56,6 +56,9 @@ module.exports = {
 Last but not least, if you've already created a separate repository and package for your loader, you could [`npm link`](https://docs.npmjs.com/cli/link) it to the project in which you'd like to test it out.
 
 
+T> Use [`webpack-default`](https://github.com/webpack-contrib/webpack-defaults) to setup your webpack loader project. It will generate a webpack's standard project for your loader
```

Comment:
```suggestion
T> Use [`webpack-default`](https://github.com/webpack-contrib/webpack-defaults) to setup your webpack loader project. It will generate a webpack standard project for your loader.
```